front/src/applications/operationalStudies/Home.tsx

```diff
@@ -16,7 +16,7 @@ import {
   osrdEditoastApi,
 } from 'common/api/osrdEditoastApi';
 import { useSelector } from 'react-redux';
-import { RootState } from 'reducers';
+import { getUserPreferences } from 'reducers/user/userSelectors';
 
```

Comment:
Right now, userPreference object has only one property, ok. But it is a good habit not to decompose object, otherwise the component would be rerendered on any userPreference update, whereas you only need safeWord.

Create specific selectors

Reply:
I add the getUserSafeWord action to get only the safweWord from userPreferences